tools/release.sh

```diff
@@ -136,7 +127,7 @@ pip install \
   --no-cache-dir \
   --extra-index-url http://localhost:$PORT \
   --constraint ../constraints.txt \
-  josepy[tests]
+  josepy
```

Comment:
Now when I run this script, this line crashes with:
```
ERROR: In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
    josepy from http://localhost:1234/josepy/josepy-99.0.0-py3-none-any.whl
```
It'd be nice to somehow add josepy's hash here, but barring you having some elegant idea on how to do this, I think we should run `poetry export` with `--without-hashes`.

Reply:
Well, in theory we can just calculate the hash ourselves from the assets generated by `poetry build`, then expose these hashes in the local pypi server. But it sounds like a lot of complexity for not that much of gains. Very unfortunate that pip do not support (yet?) a mix of hashes/unhashes dependencies.

I guess the best for now is to remove the other hashes. I do that.